deploy/canary/Dockerfile

```diff
@@ -3,8 +3,7 @@ MAINTAINER vmarmol@google.com
 
 RUN apt-get install -y git dmsetup
 RUN git clone https://github.com/google/cadvisor.git /go/src/github.com/google/cadvisor
-RUN go get github.com/tools/godep
-RUN cd /go/src/github.com/google/cadvisor && godep go build .
+RUN cd /go/src/github.com/google/cadvisor && make
```

Comment:
ditto: use `make build` here. Are you sure the Makefile should be the source of truth for all our build commands? I'm inclined to prefer calling `build/build.sh` directly, but don't have any concrete reasons for it.